### PR TITLE
Add terminate command

### PR DESCRIPTION
**Pull request by paulgb**

### CODE REVIEW

File: src/commands/terminate.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Command, Flags } from '@oclif/core'
+import { Jamsocket } from '../jamsocket'
+
+const MAX_PORT = (2 ** 16) - 1
```
Comment (rolyatmax) [inline src/commands/terminate.ts]: I think we can drop this

File: src/api.ts [hunk 2]
```diff
@@ -67,6 +67,8 @@ export interface SpawnTokenRevokeResult {
   status: 'ok',
 }
 
+export interface TerminateResult {}
```
Comment (rolyatmax) [inline src/api.ts]: Looks like the result is `{ status: 'ok' }`: https://github.com/drifting-in-space/p2/blob/main/jamsocket/src/platform/server/api/backend.rs#L166

File: src/commands/terminate.ts (hunk 1 above)
Comment (paulgb) [reply src/commands/terminate.ts]: oops yes, done.

File: src/api.ts (hunk 2 above)
Comment (paulgb) [reply src/api.ts]: Indeed, good catch.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jamsocket",
-  "version": "0.4.5",
+  "version": "0.4.6",
```
Comment (rolyatmax) [inline package.json:3]: We can drop this back to `0.4.5` and then use [these instructions](https://www.notion.so/drifting/Publish-CLI-to-NPM-Steps-ff8af8b19e414cee9d650ec328535263) for updating the version and publishing it to NPM.

**Review by rolyatmax — APPROVED** [review]
Aside from that last comment above, this looks good! Pulled it down, and it worked for me! Thanks for pulling this together!

**Pull request merged**